docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -115,16 +114,16 @@ Use the {product-long-api-designer} web console to create an OpenAPI or AsyncAPI
 
 ifndef::qs[]
 .Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
+* You're logged in to the {product-long-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. In the {service-url-api-designer}[{product-long-api-designer} web console^], click *Create a schema or API design*.
 . Complete the form to provide the details of the new design:
 .. *Name*: Enter a unique design name, such as `my-api-design`.
-.. *Summary*: Optional: Enter a short description of the design.
+.. *Summary*: _Optional:_ Enter a short description of the design.
 .. *Type*: Accept the default value *OpenAPI*, or select another API type from the list.
-.. *Version*: OpenAPI only: Accept the default value *3.0.2*, or select another version from the list.
+.. *Version*: _OpenAPI only:_ Accept the default value *3.0.2*, or select another version from the list.
 .. *Template*: Select the *Pet Store Example* template, or select another template from the list.
```

Comment:
Wondering if Pet Store Example should be enclosed with backticks because it is as a proper (literal) name and not a UI label?

Reply:
It's the name of the template in the UI, similar to selecting an item from a menu, so I thought bold formatting was best? It is not text that the user enters in a text field.